make-release.sh

```
@@ -0,0 +1,201 @@
+      '-v'|'--version') VERSION="$2"; shift 1;;
+      '--no-push') NO_PUSH=1; shift 0;;
+      '--skip-publish') SKIP_PUBLISH=1; shift 0;;
+      '--skip-bump-version') SKIP_NEXT_VERSION_BUMP=1; shift 0;;
+      '--help'|'-h') usage;;
+    esac
+    shift 1
+  done
+
+  [[ -z ${VERSION} ]] && { echo "[ERROR] Release version is not defined"; usage; }
+
+  X_BRANCH=$(echo "${VERSION}" | sed 's/.$/x/')
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)